addons/website/static/src/snippets/s_dynamic_snippet/000.js

```diff
@@ -195,6 +195,11 @@ const DynamicSnippet = publicWidget.Widget.extend({
             $target: $templateArea,
         });
         $templateArea.html(this.renderedContent);
+        // TODO remove me in master. These classes should be added in the XML
+        // ('website.s_dynamic_snippet.grid') rather than here.
+        for (const rowElement of $templateArea[0].querySelectorAll('.dynamic_snippet_template > .row')) {
+            rowElement.classList.add('s_col_no_resize', 's_col_no_bgcolor');
+        }
```

Comment:
I'd prefer something more generic, which is linked to the fact that the dynamic snippet content is not editable (you should not have to say it is "no resize, no bg_color": it is not editable.